docs/labelling.md

```diff
@@ -0,0 +1,69 @@
+# Labelling
+
+## Metadata labels
+
+The agent does its best to enrich profiles with labels coming from various system facilities relative to the originating process.
+
+### Profiler
+
+* `__name__`: The name of the profiler (e.g. `parca_agent_cpu`).
+* `pid`: The PID of the process.
+
+### Service Discovery
+
+#### Kubernetes
+
+* `namespace`: The namespace of the pod object.
+* `pod`: The name of the pod object.
+* `container`: The name of the container.
+* `containerid`: The ID of the container.
```

Comment:
Perhaps we can explain where the ID comes from and what it is? (e.g. is it a group id? is it v1 or v2? etc)